docs/Docker.md

```diff
@@ -0,0 +1,53 @@
+# Docker
+
+You may use docker images of the ADBench project without having to install dependencies on your PC.
+
+## Building docker image
+
+1. Clone repository
```

Comment:
Misprint: `Clone repository` instead of `Clone the repository`

Reply:
fixed